lib/src/map_widget.dart

```diff
@@ -48,7 +48,7 @@ class MapWidget extends StatefulWidget {
     // FIXME Flutter 3.x has memory leak on Android using in SurfaceView mode, see https://github.com/flutter/flutter/issues/118384
     // As a workaround default is true.
     this.textureView = true,
-    this.androidHostingMode = AndroidPlatformViewHostingMode.HC,
+    this.androidHostingMode = AndroidPlatformViewHostingMode.VD,
```

Comment:
I checked and it seems that the rendering bug with VD is gone on the latest version of Android 15 latest beta(4.2 IIRC), I feel, though, that this is a bigger change(changing global hosting mode).
How about we check that there are no issues on Android 15 release(should be available for Pixel 8 in the next couple of weeks) and ship it in the next beta(2.4.0-beta.1)?